### DISCUSSION TITLE
what are this white cubes is view ?

### DISCUSSION THREAD

**Discussion by damavand1** [introlab/rtabmap#1077] [General]
what are this white cubes is view ?
what is the benefit of this cubes ?

tnx
![Screenshot 2023-07-02 111114](https://github.com/introlab/rtabmap/assets/104978921/677a72e5-7fd8-4e52-9f6f-683c91e99278)

**Comment by matlabbe** [comment]
It is a VTK feature, they would appear when you interact with the rendering surface. When a white box appears, it means a point cloud disappeared to save time on rendering when you interact. They will disappear when you stop interacting with the surface. If you don't want to see them at all when interacting, right-click on the 3D map view, then select "Set rendering rate..." to 0 Hz.

**Reply by damavand1** [reply]
grate 
thank you for your perfect Product Rtabmap
